### ISSUE TITLE
Add the ability to control the size of thumbnail previews

### ISSUE THREAD

**Issue by hogabi-REAL** [0x90d/videoduplicatefinder#320]
**Environment**
 - OS: Any

**Describe the solution you'd like**
Not sure if this is a thing already as I can't find the option or the shortcut if there's one, but it'd be nice to be able to control the size of previews for videos/images. I'd especially like to be able to increase the size for them as it's faster to compare previews than to play video files one by one. I remember older versions of this app having little icons on the right allowing users to do that (version 1.6.0.0 had it).

Also, got a question. Is there a list of shortcuts somewhere? I feel like I might be missing things out, because I don't know them all.

**Comment by kattjevfel** [comment]
![2022-06-12_13-47-39](https://user-images.githubusercontent.com/6226312/173231657-7eba2d03-05dd-400c-bea8-5ab6951f322d.png)
There's the "Show Enlarged Thumbnail On Mouse Hover"-option, but it barely makes the thumbnail any larger and it's still so low res you still end up having to just open the video/image in an external application.

**Comment by hogabi-REAL** [comment]
> ![2022-06-12_13-47-39](https://user-images.githubusercontent.com/6226312/173231657-7eba2d03-05dd-400c-bea8-5ab6951f322d.png) There's the "Show Enlarged Thumbnail On Mouse Hover"-option, but it barely makes the thumbnail any larger and it's still so low res you still end up having to just open the video/image in an external application.

Yeah, the "Show enlarged thumbnail on mouse hover" sounds like a really great feature, but there's currently no point in it since the "enlarged thumbnail" is even smaller than the regular one, at least for me. It'd be cool if this feature worked as intended with a much bigger preview with good res on mouse hover.

**Comment by 0x90d** [comment]
> Is there a list of shortcuts somewhere

Unfortunately there is not but if someone wants to create it, go ahead. Either in readme or within the app.

**Comment by reddyshyam** [comment]
Good request and great implementation. Very helpful. Thanks @0x90d  👏 👍
